packages/react-charting/src/utilities/UtilityUnitTests.test.ts

```diff
@@ -970,180 +933,6 @@ describe('domainRageOfVerticalNumeric', () => {
   });
 });
 
-describe('getDomainNRangeValues', () => {
```

Comment:
why remove test cases?

Reply:
Removed the test cases for the functions which were removed as the original functions were invoked. The tests for the original functions they call are still there.

Reply:
Can you share the corresponding tests for the scenario that you have removed

Reply:
The functions that say 'getDomainNRangeValues' function called earlier were: domainRangeOfNumericForAreaChart
domainRangeOfVSBCNumeric
domainRageOfVerticalNumeric
domainRangeOfNumericForHorizontalBarChartWithAxis
domainRangeOfDateForAreaLineVerticalBarChart
domainRangeOfXStringAxis

All tests of these internal functions exist in the same file.

Reply:
For createYAxis: (removed), following are the internal functions whose tests are still there:
createYAxisForHorizontalBarChartWithAxis
createYAxisForOtherCharts-> renamed to createNumericYAxis

For createStringYAxis: (removed), following are the internal functions whose tests are still there:
createStringYAxisForHorizontalBarChartWithAxis
createStringYAxisForOtherCharts

For getDomainNRangeValues (removed), following are the internal functions whose tests are still there:

domainRangeOfNumericForAreaChart
domainRangeOfVSBCNumeric
domainRageOfVerticalNumeric
domainRangeOfNumericForHorizontalBarChartWithAxis
domainRangeOfDateForAreaLineVerticalBarChart
domainRangeOfXStringAxis


For getMinMaxOfYAxis(removed), following are the internal functions whose tests are still there:

findNumericMinMaxOfY
findVSBCNumericMinMaxOfY
findVerticalNumericMinMaxOfY
findHBCWANumericMinMaxOfY

Reply:
for minmax of y HeatMap case is missing
for domainNrange GroupedVerticalNumeric case is missing

Reply:
Updated tests